Formula/pilosa.rb

```diff
@@ -0,0 +1,63 @@
+class Pilosa < Formula
+  desc "Distributed bitmap index that queries across data sets"
+  homepage "https://www.pilosa.com"
+  url "https://github.com/pilosa/pilosa/archive/v0.3.1.tar.gz"
```

Comment:
This [release](https://github.com/pilosa/pilosa/releases/tag/v0.3.2) just adds Docker support.